litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/WorkflowPreview/index.tsx

```diff
@@ -1,22 +1,141 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
+import DagreGraph, { d3Link, d3Node } from '../../../../components/DagreGraph';
+import { Steps } from '../../../../models/redux/customyaml';
 import { RootState } from '../../../../redux/reducers';
 import { extractSteps } from '../ExtractSteps';
+import { createLabel } from './createLabel';
+import useStyles from './styles';
+
+interface GraphData {
+  nodes: d3Node[];
+  links: d3Link[];
+}
 
 interface WorkflowPreviewProps {
   isCustom: boolean;
```

Comment:
can isCustom be changed to a more descriptive variable

Reply:
What name would you suggest? 
This variable only denotes if the Dagre graph is for a custom workflow or not.

Reply:
how about `isCustomWorkflow` or `isUserDefinedWorkflow` ?

Reply:
Updated the variable